docs/changelog.md

```diff
@@ -0,0 +1,34 @@
+## 2.6.0
+### Minor changes
+- Added a login state button, similar to Gmail, allowing quicker re-authentication with a passkey for previously authenticated accounts.
```

Comment:
what does "similar to Gmail" mean? => am I not sure if that's clear to readers (at least to me it's not clear)